package-lock.json

```diff
@@ -1,686 +0,0 @@
-{
```

Comment:
I don't know enough about Node to know whether this package-lock file is helpful or not. I've seen several places where it's recommended to add it to version control (I guess in a similar way to pinning your python dependencies). What's your thinking behind removing it (I have no strong feelings one way or the other)?

Reply:
I don't quite remember. I guess my reasoning was that since the dependencies are declared in ``package.json`` using lower bounds on the versions, you get newer versions where available. I'm also not familiar enough with Node to answer this properly, so I'll try to do some reading. It would probably take care of all the dependabot PRs though :smile:

Reply:
Discussed in person - we think this should be OK.